main.go

```diff
@@ -10,14 +10,41 @@ import (
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
-	"github.com/google/go-github/v39/github"
+	"github.com/google/go-github/v53/github"
 	"golang.org/x/oauth2"
 )
 
+const (
+	TimeOut             = 2 * time.Second
+	initialInterval     = 800 * time.Millisecond
+	randomizationFactor = .5
+	multiplier          = 3
+	maxInterval         = 15 * time.Second
+	maxElapsedTime      = 3 * time.Minute
+	BlinkGitHubUser     = "blinkhealthgithub"
+
+	/* Time elapsed between pull request 'create time' and the time it took
+	 * Atlantis to create the comment - which is the terragrunt plan.  Multiple
+	 * comments from Atlantis __can__ exist, but we only care about the most
+	 * recent one.
+	 */
+	AcceptableTimeDelta = 65 //seconds
+)
+
 var client *github.Client
 var ctx context.Context = context.Background()
 var atlantisPath string
 
+func getPrCreatedAt(ctx context.Context, client github.Client, org string, repo string, prNum int) github.Timestamp {
+	var pr *github.PullRequest
+	var err error
+	pr, _, err = client.PullRequests.Get(ctx, org, repo, prNum)
```

Comment:
i don't think you need the explicit type declarations, and even if you want them, i think you can make this all one line

```suggestion
	pr *github.PullRequest, _, err error := client.PullRequests.Get(ctx, org, repo, prNum)
```